wuvt/static/css/wuvt.css

```diff
@@ -794,4 +811,94 @@ body > footer {
     font-size: 0.7em;
     text-align: center;
 }
+
+@media only screen and (max-width: 810px) {
+    header #mainheader #robot #vol_btn_wrapper {
+        cursor: not-allowed;
+    }
+    header #mainheader #robot #volume_btn { /*Page is too narrow for volume slider to fit*/
+        pointer-events: none;
+    }
+}
+
+@media only screen and (max-width: 990px) { /*tablet or mobile layout*/
+    header #mainheader {
+        text-align: center;
+    }
+    header #mainheader #robot #stream_btn {
+        margin-left: -41px;
+    }
+    header #mainheader #robot #volume_btn {
+        margin-left: 8px;
+        padding-left: 0px;
+	width: 43px;
+    }
+}
+
+@media only screen and (max-width: 610px) { /*mobile layout*/
+    #nowplaying {
+        background-image: none;
+	height: auto;
+	width: 100%;
+	margin: 8px 0px 8px 0px;
+	padding: 0 0 0 0;
+    }
+    #nowplaying li {
+	    white-space: normal;
```

Comment:
This was a good call, although it's not applied at tablet width so the current song can still get very cut off. A possible alternative approach would be using flexbox to dynamically size the Currently On Air box to be as large as the Current DJ box lets it (and maybe adding a max width to that for DJs with extremely long names). The previous design forcing both to be 50% seems wasteful when the DJ name is often much less than 50% wide.

Reply:
on small screens, we also eliminate the little banner tail, which is fine, but should probably be visible on tablet-sized devices and larger.

Reply:
> on small screens, we also eliminate the little banner tail, which is fine, but should probably be visible on tablet-sized devices and larger.

banner is still visible on tablet view